src/vs/workbench/contrib/positronPlots/browser/components/actionBars.tsx

```diff
@@ -179,16 +180,11 @@ export const ActionBars = (props: PropsWithChildren<ActionBarsProps>) => {
 							: null
 						}
 						{enableEditorPlot ?
-							<ActionBarButton
-								iconId='go-to-file'
-								align='right'
-								tooltip={localize('positron-open-plot-editor', "Open plot in editor")}
-								ariaLabel={localize('positron-open-plot-editor', "Open plot in editor")}
```

Comment:
I think we may have lost the ariaLabels on the `Open plot in [location]` elements